assets/chezmoi.io/docs/reference/commands/secret.md

```diff
@@ -45,6 +45,15 @@ Name of the user.
 
 New value.
 
+## Notes
```

Comment:
Another consistency question here. I appreciate that this has been moved above `## Example` because of consistency, but should `## Notes` be *before* or *after* `## Examples`? I don't think that there's a *correct* choice here (some of the notes should be understood before you run a particular command; others don't matter when you read them — most people aren't on FreeBSD).

Reply:
This one I moved for consistency, yes. I don't have strong preference on which one should come first. There are not many notes here, but most commands have examples. Let's move notes to the end?